src/App.tsx

```diff
@@ -1,20 +1,35 @@
-import Logo from "@assets/logo.png";
-import DataContainer from "@components/DataContainer/DataContainer";
-import HelloWorld from "@components/HelloWorld/HelloWorld";
+// import Logo from "@assets/logo.png";
+// import DataContainer from "@components/DataContainer/DataContainer";
+// import HelloWorld from "@components/HelloWorld/HelloWorld";
 
-import styles from "./App.module.css";
+// import styles from "./App.module.css";
 
-export default function App() {
+// export default function App() {
+//   return (
+//     <main className={styles.main}>
+//       <img className={styles.logo} alt="React logo" width="400px" src={Logo} />
+//       <DataContainer />
+//       <div>
+//         {`VITE_SOME_ENVIRONMENT_VARIABLE from .env.local: "${
+//           import.meta.env.VITE_SOME_ENVIRONMENT_VARIABLE
+//         }"`}
+//       </div>
+//       <HelloWorld msg="Hello EE Practicum Team 2!" />
+//     </main>
+//   );
+// }
```

Comment:
Don't be afraid to delete! That's what's good about version control -- we can always look at past versions if we need to add something back in.